projects/prison-custody-status-to-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/institution/Institution.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.hmpps.integrations.delius.institution
+
+import org.hibernate.annotations.Immutable
+import org.hibernate.annotations.Type
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Immutable
+@Entity
+@Table(name = "r_institution")
+class Institution(
+    @Id
+    @Column(name = "institution_id")
+    val id: Long,
+
+    @Column
+    val code: String,
+
+    @Column(name = "nomis_cde_code", columnDefinition = "CHAR(6)")
+    val nomisCdeCode: String,
+
+    @Column
+    @Type(type = "yes_no")
+    var establishment: Boolean,
+
+    @Column
+    @Type(type = "yes_no")
+    val selectable: Boolean,
```

Comment:
I was wary of removing this in case there was a duplicate code, but with only one active.  Checking in pre-prod though, that doesn't look to be the case currently (as long as we're note passing null):
```
> select count(*), nomis_cde_code from r_institution group by nomis_cde_code having count(*) > 1;
| COUNT(*) | NOMIS_CDE_CODE |
| 389	   | (null)         |
```

So happy to remove this if you'd prefer?  We just need to make sure they don't ever de-activate one and create a new one with the same code.

Reply:
Okay will leave for now